Sources/UInt128.swift

```diff
@@ -806,12 +679,31 @@ extension UnsignedInteger {
 // MARK: - Extend String for UInt128
 extension String {
     public init(_ value: UInt128) {
-        self.init()
-        self.append(value.toString())
+        self = value.toString()
     }
     public init(_ value: UInt128, radix: Int, uppercase: Bool = true) {
-        self.init()
-        let string = value.toString(radix: radix, uppercase: uppercase)
-        self.append(string)
+        self = value.toString(radix: radix, uppercase: uppercase)
+    }
+
+    var radix : UInt8 {
```

Comment:
👎 

Since this framework is rather low level, I'm extremely reticent to add new properties to built in types. Anyone who uses this library would have this property available on UInt8 types, and this could be unexpected/unwelcomed. I would prefer to keep this logic where it already is instead of extending another type with it.